composeApp/src/commonMain/kotlin/com/github/familyvault/backend/client/IPrivMxClient.kt

```diff
@@ -20,7 +20,12 @@ interface IPrivMxClient {
         type: String,
         name: String
     ): String
-
+    fun updateThread(
+        threadId: String,
+        users: List<PrivMxUser>,
+        managers: List<PrivMxUser>,
+        newName: String? = ""
```

Comment:
Jeżeli chcemy aby brak podania `newName` powodował brak jego aktualizacji, to powinno być bardziej:
```kotlin
newName: String? = null
```